packages/web-helpers/src/WebUser.ts

```diff
@@ -171,12 +222,14 @@ export class WebUser extends EventEmitter {
                 console.log(`Account connected with did: ${did}`)
             }
 
+            const profile = await this.getPublicProfile()
+
             this.account = account
-            this.context = context!
+            this.context = context
             this.did = did
 
-            const profile = await this.getPublicProfile()
-            this.client = context!.getClient()
+            // Question: Do we have to get overwrite the client created initially in the constructor by the client from context? (knowing we created the context from this client while connecting)
```

Comment:
Yes, creating the context uses the account object which is then used to create a client that contains that account object.

Reply:
Oh, ok, because `Network.connect` which takes the config of the Client, not a Client instance